app/views/homes/show.html.erb

```diff
@@ -1,18 +1,22 @@
-<h1><%= title %></h1>
+<% 6.times do %>
+  <h1><%= title %></h1>
```

Comment:
Ideally we'd have one h1 per page. But in this case, every h1 has the same text. So I think it's OK?

I welcome opinions.